ruby-challenges.rb

```diff
@@ -8,12 +8,50 @@
 # Expected output: ['Arizona', 'California', 'Idaho', 'Maine', 'Nevada', 'New Hampshire', 'Oregon', 'Rhode Island', 'Washington'] 
 
 # Pseudo code:
+#Define a method sort_states
+#Extract the values of the array onto a single array
+    #Use .flatten
+#sort the array
+    #Use .sort
+#return the sorted array
+
+def sort_states(hash)
+    combined_sorted_array = hash.values.flatten.sort
```

Comment:
Nice work!  You can simplify this by removing the assigned variables and just returning the flattened/sorted hash.